src/includes/performance/retrieve-transaction/php.mdx

```diff
@@ -0,0 +1,54 @@
+## Setting and Retrieving the Current Transaction
+
+In cases where you want to attach spans to an already ongoing transaction you can use `getTransaction()` on the current hub. This property will return a `\Sentry\Tracing\Transaction` in case there is a transaction set otherwise it returns `null`.
+
+```php
+$transaction = \Sentry\SentrySdk::getCurrentHub()->getTransaction();
+
+if ($transaction !== null) {
+    $context = new \Sentry\Tracing\SpanContext();
+    $context->setOp('some_operation');
+    $span = $transaction->startChild($context);
+
+    // ...
+
+    $span->finish();
+}
+```
+
+You can also set the current transaction after you started one:
+
+```php
+$transactionContext = new \Sentry\Tracing\TransactionContext();
+$transactionContext->setName('Example Transaction');
+$transactionContext->setOp('http.request');
+
+$transaction = \Sentry\startTransaction($transactionContext);
+
+// A transaction is a span so we set it using `setSpan`
+\Sentry\SentrySdk::getCurrentHub()->setSpan($transaction);
+```
+
+## Setting and Retrieving the Current Span
+
+In cases where you want to attach spans to an already ongoing span you can use `getSpan()` on the current hub. This property will return a `\Sentry\Tracing\Span` in case there is a span set otherwise it returns `null`. You can also set the current span after you started one and restore it back to the parent after your span ended.
```

Comment:
```suggestion
In cases where you want to attach spans to an already ongoing span, you can use `getSpan()` on the current hub. This property will return a `\Sentry\Tracing\Span` in cases where there is a span set; otherwise, it returns `null`. You can also set the current span after you started one and restore it back to the parent after your span ends.
```